### DISCUSSION TITLE
`conda install numpy` won't finish forever

### DISCUSSION THREAD

**Discussion by rsuwa** [praveen-palanisamy/macad-agents#17] [Q&A]
<https://github.com/praveen-palanisamy/macad-agents/blob/b2726b391ee4f6844e2ae80883fb99f8027d34b4/Dockerfile#L103>

```bash
$ docker build --rm -f macad-agents/Dockerfile -t macad-agents:latest .
...

Step 8/15 : RUN conda install -y numpy
 ---> Running in 7123a03249d8
Solving environment: ...working...
```

No matter how many hours I wait, it stops as described above.

**Comment by rsuwa** [comment]
```bash
Step 8/15 : RUN conda install -y numpy
 ---> Running in 8939ec80c4bb
Solving environment: ...working... Killed
```

Finally, the process has been killed.

**Reply by praveen-palanisamy** [reply]
Hi @rsuwa ,
Are you building this Docker image locally or using a build service?
Although the the conda install operation can take some time, it will complete unless interrupted by the system for example, due to a time limit by the build system.

**Reply by Panshark** [reply]
I have the same problem when building the Docker image locally. I tried twice, it always got stuck in here:

```
Step 8/15 : RUN conda install -y numpy

 ---> Running in 0ae7ce97b684

Solving environment: ...working...
```
